sql/moz-fx-data-shared-prod/firefox_desktop_derived/report_content_live_v1/metadata.yaml

```diff
@@ -0,0 +1,43 @@
+friendly_name: Report Content Live
+description: |-
+  Reported organic content using the live tables.
+
+  For each report, we collect
+    submission date
+    card type
+    corpus item id
+    report reason
+    section
+    section position
+    title
+    topic
+    url
+owners:
+- lmcfall@mozilla.com
+labels:
+  incremental: true
+  schedule: hourly
+  shredder_mitigation: true
+scheduling:
+  dag_name: bqetl_report_content_hourly
+  date_partition_parameter: null
+  # We reprocess the same day every hour up until 1:00 the following day, to give
+  # the live data time to come in
+  destination_table: >-
+    report_content_live_v1${{
+    (execution_date - macros.timedelta(hours=1)).strftime("%Y%m%d")
+    }}
+  parameters:
+  - >-
+    submission_date:DATE:{{
+    (execution_date - macros.timedelta(hours=1)).strftime('%Y-%m-%d')
+    }}
+  query_file_path:
```

Comment:
I'm not quite sure I understand why we need this.

Reply:
I double-checked, and it looks like setting `query_file_path` in this scenario isn't required, courtesy of [an improvement Anna made way back in 2022](https://github.com/mozilla/telemetry-airflow/pull/1525/files#r91025205).

Reply:
I double-checked, and it looks like setting `query_file_path` in this scenario isn't required, courtesy of [an improvement Anna made way back in 2022](https://github.com/mozilla/telemetry-airflow/pull/1525/files#r91025205).